studio/src/pages/[organizationSlug]/apikeys.tsx

```diff
@@ -490,7 +498,14 @@ const CreateAPIKeyDialog = ({
           <Button
             className="mt-2"
             type="submit"
-            disabled={!isValid || !!errorMsg}
+            disabled={
+              !isValid ||
+              !!errorMsg ||
+              (rbac &&
```

Comment:
Can we add a comment here about the right behaviour?

Reply:
Done